packages/form-js-editor/test/helper/mocks/index.js

```diff
@@ -0,0 +1,165 @@
+
```

Comment:
I like the efforts to harmonize our mocking strategy 🚀 @vsgoulart probably I good thing for you to have a look at to gather the general approach of mocking bpmn-io services. 

I wonder whether we need some adjustments in the Playground.spec as we have errors over there 

![image](https://github.com/bpmn-io/form-js/assets/9433996/dbf4c065-cba3-4272-af24-f83e74f13c78)